README.md

```diff
@@ -1,25 +1,120 @@
 # Indico Tile Server (CERN)
 
-This is a simple tile server microservice that we are publishing in hopes of being helpful to others who want to use the map functionality in [Indico 2.2](https://github.com/indico/indico) without a commercial provider.
+This is a simple tile server microservice (container) that we are publishing in hopes of being helpful to others who want to use the map functionality in [Indico 2.2](https://github.com/indico/indico) without a commercial provider.
```

Comment:
not sure if that fits here - while deploying as a container is probably the easiest way, nothing stops you from deploying it the "classic" way without docker